dags/glam_fenix.py

```diff
@@ -9,23 +9,28 @@
     "owner": "amiyaguchi@mozilla.com",
     "depends_on_past": False,
     "start_date": datetime(2020, 2, 19),
-    "email": ["telemetry-alerts@mozilla.com", "amiyaguchi@mozilla.com", "bewu@mozilla.com"],
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "amiyaguchi@mozilla.com",
+        "bewu@mozilla.com",
+    ],
     "email_on_failure": True,
     "email_on_retry": True,
     "retries": 1,
     "retry_delay": timedelta(minutes=30),
 }
 
+# Fenix as a product has a convoluted app_id history.
+# https://docs.google.com/spreadsheets/d/18PzkzZxdpFl23__-CIO735NumYDqu7jHpqllo0sBbPA
 PRODUCTS = [
-    "org_mozilla_fenix",
-    "org_mozilla_firefox",
-    "org_mozilla_firefox_beta",
-    "org_mozilla_fennec_aurora",
+    "org_mozilla_fenix",  # 2019-07-03 - present (nightly)
+    "org_mozilla_fenix_nightly",  # 2019-06-30 - 2020-07-03
+    "org_mozilla_firefox",  # 2020-07-28 - present
+    "org_mozilla_firefox_beta",  # 2020-03-26 - present
+    "org_mozilla_fennec_aurora", # 2020-01-21 - present
```

Comment:
What do the dates mean?  When they started sending glean pings?  Shouldn't fennec_aurora be in the same situation as fenix_nightly since they're both no longer listed on the app store and messaging was sent out to switch to the other apps.  In any case, it might be useful to note that fennec_aurora is the deprecated nightly.

Reply:
Oh wait nvm I see the dates

Reply:
The dates are the first appearance in the app store, which I assume is the earliest date where we'd be collecting data from an actual user population. As for fennec aurora, I misread the column for name end date being the same as app store end date, so it's correct that fenix nightly and fennec aurora are in the same boat.